InnerEye/ML/deep_learning_config.py

```diff
@@ -608,6 +609,57 @@ def get_path_to_checkpoint(self, epoch: int, for_mean_teacher_model: bool = Fals
                / self.checkpoint_folder \
                / f"{epoch}{filename}"
 
+    def get_recovery_path_train(self, run_recovery: Optional[RunRecovery],
+                                is_mean_teacher: bool, epoch: int) -> Path:
+        """
+        Decides the checkpoint path to use for the current training run. If a run recovery object is used, use the
+        checkpoint from there, otherwise use the checkpoints from the current run.
+        :param run_recovery: Optional run recovery object
+        :param is_mean_teacher: If this a mean teacher model.
+        :param epoch: Epoch to recover
+        :return: Constructed checkpoint path to recover from.
+        """
+        if run_recovery:
+            checkpoint_paths = run_recovery.get_checkpoint_paths(epoch, is_mean_teacher)[0]
+        else:
+            checkpoint_paths = self.get_path_to_checkpoint(epoch, is_mean_teacher)
```

Comment:
This line is a remnant of the code before the refactor, but is there any situation this would be needed? Seems to look for the checkpoint for the starting epoch, which will never exist. The only time this function is called is at the beginning of the training code.